src/dtgtk/range.h

```diff
@@ -177,6 +179,7 @@ void dtgtk_range_select_redraw(GtkDarktableRangeSelect *range);
 
 // get a human readable text for bounds
 gchar *dtgtk_range_select_get_bounds_pretty(GtkDarktableRangeSelect *range);
+gchar *dtgtk_range_select_get_rating_bounds_pretty(GtkDarktableRangeSelect *range);
```

Comment:
I think that this can go in libs/filters/rating.c as as static func...

Reply:
moved